docs/source/markdown/guides/developer/contributing.md

```diff
@@ -0,0 +1,97 @@
+# Contribution Guidelines
```

Comment:
Do we need this here? Can't we point to the contribution guideline we have on the root of the repo?